packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -32,69 +35,56 @@ $tc-collapsible-panel-btn-color: #555964;
 	}
 
 	&.col-6 {
-		@include type-col(calc(100%/6));
+		@include type-col(calc(100% / 6));
 	}
 
 	&.col-7 {
-		@include type-col(calc(100%/7));
+		@include type-col(calc(100% / 7));
 	}
 
 	&.col-8 {
-		@include type-col(calc(100%/8));
+		@include type-col(calc(100% / 8));
 	}
 
 	&.col-9 {
-		@include type-col(calc(100%/9));
+		@include type-col(calc(100% / 9));
 	}
 
 	&.col-10 {
 		@include type-col(calc(10%));
 	}
 }
+
 @mixin header-style {
 	display: flex;
 	align-items: center;
-	justify-content: space-between;
-	padding-right: $tc-collapsible-panel-padding-larger;
 
 	> * {
 		@include col-style;
 	}
 }
 
-.group {
-	justify-content: space-between;
-
-	> * {
-		margin: 0 $tc-collapsible-panel-padding-normal;
-	}
-
-	:global(.label) {
-		white-space: nowrap;
-		text-overflow: ellipsis;
-		overflow: hidden;
-		font-size: 1em;
-	}
-}
-
-.tc-panel {
+.tc-default-no-content-panel {
 	:global(.panel-heading) {
-		@include header-style;
-		padding-right: $tc-collapsible-panel-padding-larger + $tc-collapsible-panel-padding-normal;
+		padding: 0 5px 0 0;
```

Comment:
In order to constrain the user to toggle only on the caret, the user should not have another place where to click to fold/unfold the panel.